References/DNN/09.11.00/DotNetNuke.DependencyInjection.deps.json

```diff
@@ -74,12 +74,12 @@
       "serviceable": false,
       "sha512": ""
     },
-    "Microsoft.Build.Tasks.Git/1.0.0": {
+    "Microsoft.Build.Tasks.Git/1.1.1": {
       "type": "package",
       "serviceable": true,
-      "sha512": "sha512-z2fpmmt+1Jfl+ZnBki9nSP08S1/tbEOxFdsK1rSR+LBehIJz1Xv9/6qOOoGNqlwnAGGVGis1Oj6S8Kt9COEYlQ==",
-      "path": "microsoft.build.tasks.git/1.0.0",
-      "hashPath": "microsoft.build.tasks.git.1.0.0.nupkg.sha512"
+      "sha512": "sha512-AT3HlgTjsqHnWpBHSNeR0KxbLZD7bztlZVj7I8vgeYG9SYqbeFGh0TM/KVtC6fg53nrWHl3VfZFvb5BiQFcY6Q==",
```

Comment:
@ArielBlanco1990 Should this be committed? 🤔

Reply:
@WillStrohl I don't think it should be in the commit, I'll delete it, it was inside the DNN 09.11.00 folder and I didn't notice it.

Reply:
@ArielBlanco1990 I don't see a commit that addresses this.  🤔